cmd/2016_1/main_test.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/afarbos/aoc/pkg/point"
+	"github.com/afarbos/aoc/pkg/test"
+	"github.com/afarbos/aoc/pkg/utils"
+)
+
+func TestBlockDistance(t *testing.T) {
+	const (
+		sequences1 = "R2, L3"
+		sequences2 = "R2, R2, R2"
+		sequences3 = "R5, L5, R5, R3"
+	)
+
+	testData := map[string]int{
+		sequences1: 5,
+		sequences2: 2,
+		sequences3: 12,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '12', create a named constant for it](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498955)